plainly-plugin/src/node/collect/index.ts

```diff
@@ -1 +1,18 @@
+import path from 'path';
+
+import { evalScriptAsync } from '../utils';
+import { collectProjectFiles, zip } from './collect';
+
 export * from './collect';
+
+export const makeProjectZip = async (targetPath: string) => {
```

Comment:
why is this separated in `index.ts`? just have it in collect, it's anyway that functionality?